tree/dataframe/src/RDFHelpers.cxx

```diff
@@ -48,14 +51,39 @@ void ROOT::RDF::RunGraphs(std::vector<RResultHandle> handles)
    std::set<RResultHandle, decltype(sameGraph)> s(handles.begin(), handles.end(), sameGraph);
    std::vector<RResultHandle> uniqueLoops(s.begin(), s.end());
 
+   // Trigger jitting. One call is enough to jit the code required by all computation graphs.
+   TStopwatch sw;
+   sw.Start();
+   {
+      auto silenceRDFLogs = ROOT::Experimental::RLogScopedVerbosity(ROOT::Detail::RDF::RDFLogChannel(),
+                                                                    ROOT::Experimental::ELogLevel::kError);
+      uniqueLoops[0].fLoopManager->Jit();
+   }
+   sw.Stop();
+   R__LOG_INFO(ROOT::Detail::RDF::RDFLogChannel())
+      << "Just-in-time compilation phase for RunGraphs (" << uniqueLoops.size()
+      << " unique computation graphs) completed"
+      << (sw.RealTime() > 1e-3 ? " in " + std::to_string(sw.RealTime()) + " seconds." : " in less than 1ms.");
```

Comment:
I couldn't find where the precision of TStopwatch is defined to be 1 ms. If this is true in general, we should probably change the logging sentence also in  `RLoopManager::Jit`

Reply:
it is since a few days